Cannot build OpenCV with Libtiff for iOS

##### System information (version)
- OpenCV => latest commit on 3.4 branch
- Operating System / Platform => MacOS 10.14.3
- Compiler => Apple clang 10.0.0.10001145 (xCode 10.1)
- CMake 3.13.4

##### Detailed description

Cannot build LIB TIFF for iOS. (with BUILD_TIFF = OFF the build runs fine)

##### Steps to reproduce
- set  BUILD_TIFF = ON in CMakeLists.txt
- run build_framework.py

------

<details>

```
...
-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- The CXX compiler identification is AppleClang 10.0.0.10001145
-- The C compiler identification is AppleClang 10.0.0.10001145
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test HAVE_CXX11 (check file: cmake/checks/cxx11.cpp)
-- Performing Test HAVE_CXX11 - Failed
-- Performing Test HAVE_CXX_FSIGNED_CHAR
-- Performing Test HAVE_CXX_FSIGNED_CHAR - Success
-- Performing Test HAVE_C_FSIGNED_CHAR
-- Performing Test HAVE_C_FSIGNED_CHAR - Success
-- Performing Test HAVE_CXX_W
-- Performing Test HAVE_CXX_W - Success
-- Performing Test HAVE_C_W
-- Performing Test HAVE_C_W - Success
-- Performing Test HAVE_CXX_WALL
-- Performing Test HAVE_CXX_WALL - Success
-- Performing Test HAVE_C_WALL
-- Performing Test HAVE_C_WALL - Success
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_C_WERROR_RETURN_TYPE
-- Performing Test HAVE_C_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WERROR_ADDRESS
-- Performing Test HAVE_CXX_WERROR_ADDRESS - Success
-- Performing Test HAVE_C_WERROR_ADDRESS
-- Performing Test HAVE_C_WERROR_ADDRESS - Success
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_CXX_WFORMAT
-- Performing Test HAVE_CXX_WFORMAT - Success
-- Performing Test HAVE_C_WFORMAT
-- Performing Test HAVE_C_WFORMAT - Success
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WMISSING_DECLARATIONS
-- Performing Test HAVE_C_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WMISSING_PROTOTYPES
-- Performing Test HAVE_C_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WUNDEF
-- Performing Test HAVE_CXX_WUNDEF - Success
-- Performing Test HAVE_C_WUNDEF
-- Performing Test HAVE_C_WUNDEF - Success
-- Performing Test HAVE_CXX_WINIT_SELF
-- Performing Test HAVE_CXX_WINIT_SELF - Success
-- Performing Test HAVE_C_WINIT_SELF
-- Performing Test HAVE_C_WINIT_SELF - Success
-- Performing Test HAVE_CXX_WPOINTER_ARITH
-- Performing Test HAVE_CXX_WPOINTER_ARITH - Success
-- Performing Test HAVE_C_WPOINTER_ARITH
-- Performing Test HAVE_C_WPOINTER_ARITH - Success
-- Performing Test HAVE_CXX_WSHADOW
-- Performing Test HAVE_CXX_WSHADOW - Success
-- Performing Test HAVE_C_WSHADOW
-- Performing Test HAVE_C_WSHADOW - Success
-- Performing Test HAVE_CXX_WSIGN_PROMO
-- Performing Test HAVE_CXX_WSIGN_PROMO - Success
-- Performing Test HAVE_C_WSIGN_PROMO
-- Performing Test HAVE_C_WSIGN_PROMO - Success
-- Performing Test HAVE_CXX_WUNINITIALIZED
-- Performing Test HAVE_CXX_WUNINITIALIZED - Success
-- Performing Test HAVE_C_WUNINITIALIZED
-- Performing Test HAVE_C_WUNINITIALIZED - Success
-- Performing Test HAVE_CXX_WNO_NARROWING
-- Performing Test HAVE_CXX_WNO_NARROWING - Success
-- Performing Test HAVE_C_WNO_NARROWING
-- Performing Test HAVE_C_WNO_NARROWING - Success
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_CXX_WNO_COMMENT
-- Performing Test HAVE_CXX_WNO_COMMENT - Success
-- Performing Test HAVE_C_WNO_COMMENT
-- Performing Test HAVE_C_WNO_COMMENT - Success
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_CXX_FLTO
-- Performing Test HAVE_CXX_FLTO - Success
-- Performing Test HAVE_C_FLTO
-- Performing Test HAVE_C_FLTO - Success
-- Performing Test HAVE_CPU_NEON_SUPPORT (check file: cmake/checks/cpu_neon.cpp)
-- Performing Test HAVE_CPU_NEON_SUPPORT - Success
-- Performing Test HAVE_CPU_FP16_SUPPORT (check file: cmake/checks/cpu_fp16.cpp)
-- Performing Test HAVE_CPU_FP16_SUPPORT - Success
-- Performing Test HAVE_CPU_BASELINE_FLAGS
-- Performing Test HAVE_CPU_BASELINE_FLAGS - Success
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE - Failed
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE - Failed
-- Looking for posix_memalign
-- Looking for posix_memalign - found
-- Looking for malloc.h
-- Looking for malloc.h - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of off64_t
-- Check size of off64_t - failed
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32 - Success
-- Performing Test HAVE_C_WNO_ATTRIBUTES
-- Performing Test HAVE_C_WNO_ATTRIBUTES - Success
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE - Success
-- Performing Test HAVE_C_WNO_UNDEF
-- Performing Test HAVE_C_WNO_UNDEF - Success
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH - Success
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for dlfcn.h
-- Looking for dlfcn.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for io.h
-- Looking for io.h - not found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for search.h
-- Looking for search.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for sys/time.h
-- Looking for sys/time.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Performing Test C_HAS_inline
-- Performing Test C_HAS_inline - Success
-- Check size of signed short
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_b116b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_b116b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_b116b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_b116b.app/cmTC_b116b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_b116b

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:64 (check_type_size)


-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_9898f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_9898f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_9898f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_9898f.app/cmTC_9898f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_9898f

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:65 (check_type_size)


-- Check size of signed int
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_676cc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_676cc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_676cc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_676cc.app/cmTC_676cc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_676cc

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:66 (check_type_size)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_ec04c
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_ec04c
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ec04c
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ec04c.app/cmTC_ec04c
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_ec04c

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:67 (check_type_size)


-- Check size of signed long
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_6a71f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_6a71f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_6a71f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_6a71f.app/cmTC_6a71f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_6a71f

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:68 (check_type_size)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_bd751
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_bd751
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_bd751
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_bd751.app/cmTC_bd751
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_bd751

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:69 (check_type_size)


-- Check size of signed long long
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_c9ca0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_c9ca0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_c9ca0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_c9ca0.app/cmTC_c9ca0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_c9ca0

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:70 (check_type_size)


-- Check size of unsigned long long
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_69e57
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_69e57
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_69e57
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_69e57.app/cmTC_69e57
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_69e57

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:71 (check_type_size)


-- Check size of unsigned char *
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_CHAR_P.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_f86fc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_f86fc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f86fc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f86fc.app/cmTC_f86fc
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_f86fc

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:72 (check_type_size)


-- Check size of size_t
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIZE_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_6999a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_6999a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_6999a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_6999a.app/cmTC_6999a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_6999a

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:76 (check_type_size)


-- Check size of ptrdiff_t
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_PTRDIFF_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_50679
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_50679
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_50679
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_50679.app/cmTC_50679
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_50679

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:77 (check_type_size)


-- Check size of INT8
-- Check size of INT8 - failed
-- Check size of INT16
-- Check size of INT16 - failed
-- Check size of INT32
-- Check size of INT32 - failed
-- Looking for floor
-- Looking for floor - found
-- Looking for pow
-- Looking for pow - found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for isascii
-- Looking for isascii - found
-- Looking for memset
-- Looking for memset - found
-- Looking for mmap
-- Looking for mmap - found
-- Looking for getopt
-- Looking for getopt - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for setmode
-- Looking for setmode - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for lfind
-- Looking for lfind - found
-- Performing Test HAVE_SNPRINTF
-- Performing Test HAVE_SNPRINTF - Success
-- Check if the system is big endian
-- Searching 16 bit integer
-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_aa23b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_aa23b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_aa23b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_aa23b.app/cmTC_aa23b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_aa23b

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/TestBigEndian.cmake:32 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_2b19b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_2b19b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_2b19b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_2b19b.app/cmTC_2b19b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_2b19b

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/TestBigEndian.cmake:37 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:116 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_3bd56
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_3bd56
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_3bd56
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_3bd56.app/cmTC_3bd56
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_3bd56

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/CheckTypeSize.cmake:241 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/TestBigEndian.cmake:44 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


CMake Error at /usr/local/Cellar/cmake/3.13.4/share/cmake/Modules/TestBigEndian.cmake:49 (message):
  no suitable type found
Call Stack (most recent call first):
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Configuring incomplete, errors occurred!
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeOutput.log".
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeError.log".
```

</details>

Cmake 3.10.0

-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- The CXX compiler identification is AppleClang 10.0.0.10001145
-- The C compiler identification is AppleClang 10.0.0.10001145
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test HAVE_CXX11 (check file: cmake/checks/cxx11.cpp)
-- Performing Test HAVE_CXX11 - Failed
-- Performing Test HAVE_CXX_FSIGNED_CHAR
-- Performing Test HAVE_CXX_FSIGNED_CHAR - Success
-- Performing Test HAVE_C_FSIGNED_CHAR
-- Performing Test HAVE_C_FSIGNED_CHAR - Success
-- Performing Test HAVE_CXX_W
-- Performing Test HAVE_CXX_W - Success
-- Performing Test HAVE_C_W
-- Performing Test HAVE_C_W - Success
-- Performing Test HAVE_CXX_WALL
-- Performing Test HAVE_CXX_WALL - Success
-- Performing Test HAVE_C_WALL
-- Performing Test HAVE_C_WALL - Success
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_C_WERROR_RETURN_TYPE
-- Performing Test HAVE_C_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WERROR_ADDRESS
-- Performing Test HAVE_CXX_WERROR_ADDRESS - Success
-- Performing Test HAVE_C_WERROR_ADDRESS
-- Performing Test HAVE_C_WERROR_ADDRESS - Success
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_CXX_WFORMAT
-- Performing Test HAVE_CXX_WFORMAT - Success
-- Performing Test HAVE_C_WFORMAT
-- Performing Test HAVE_C_WFORMAT - Success
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WMISSING_DECLARATIONS
-- Performing Test HAVE_C_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WMISSING_PROTOTYPES
-- Performing Test HAVE_C_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WUNDEF
-- Performing Test HAVE_CXX_WUNDEF - Success
-- Performing Test HAVE_C_WUNDEF
-- Performing Test HAVE_C_WUNDEF - Success
-- Performing Test HAVE_CXX_WINIT_SELF
-- Performing Test HAVE_CXX_WINIT_SELF - Success
-- Performing Test HAVE_C_WINIT_SELF
-- Performing Test HAVE_C_WINIT_SELF - Success
-- Performing Test HAVE_CXX_WPOINTER_ARITH
-- Performing Test HAVE_CXX_WPOINTER_ARITH - Success
-- Performing Test HAVE_C_WPOINTER_ARITH
-- Performing Test HAVE_C_WPOINTER_ARITH - Success
-- Performing Test HAVE_CXX_WSHADOW
-- Performing Test HAVE_CXX_WSHADOW - Success
-- Performing Test HAVE_C_WSHADOW
-- Performing Test HAVE_C_WSHADOW - Success
-- Performing Test HAVE_CXX_WSIGN_PROMO
-- Performing Test HAVE_CXX_WSIGN_PROMO - Success
-- Performing Test HAVE_C_WSIGN_PROMO
-- Performing Test HAVE_C_WSIGN_PROMO - Success
-- Performing Test HAVE_CXX_WUNINITIALIZED
-- Performing Test HAVE_CXX_WUNINITIALIZED - Success
-- Performing Test HAVE_C_WUNINITIALIZED
-- Performing Test HAVE_C_WUNINITIALIZED - Success
-- Performing Test HAVE_CXX_WNO_NARROWING
-- Performing Test HAVE_CXX_WNO_NARROWING - Success
-- Performing Test HAVE_C_WNO_NARROWING
-- Performing Test HAVE_C_WNO_NARROWING - Success
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_CXX_WNO_COMMENT
-- Performing Test HAVE_CXX_WNO_COMMENT - Success
-- Performing Test HAVE_C_WNO_COMMENT
-- Performing Test HAVE_C_WNO_COMMENT - Success
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_CXX_FLTO
-- Performing Test HAVE_CXX_FLTO - Success
-- Performing Test HAVE_C_FLTO
-- Performing Test HAVE_C_FLTO - Success
-- Performing Test HAVE_CPU_NEON_SUPPORT (check file: cmake/checks/cpu_neon.cpp)
-- Performing Test HAVE_CPU_NEON_SUPPORT - Success
-- Performing Test HAVE_CPU_FP16_SUPPORT (check file: cmake/checks/cpu_fp16.cpp)
-- Performing Test HAVE_CPU_FP16_SUPPORT - Success
-- Performing Test HAVE_CPU_BASELINE_FLAGS
-- Performing Test HAVE_CPU_BASELINE_FLAGS - Success
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE - Failed
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE - Failed
-- Looking for posix_memalign
-- Looking for posix_memalign - found
-- Looking for malloc.h
-- Looking for malloc.h - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of off64_t
-- Check size of off64_t - failed
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32 - Success
-- Performing Test HAVE_C_WNO_ATTRIBUTES
-- Performing Test HAVE_C_WNO_ATTRIBUTES - Success
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE - Success
-- Performing Test HAVE_C_WNO_UNDEF
-- Performing Test HAVE_C_WNO_UNDEF - Success
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH - Success
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for dlfcn.h
-- Looking for dlfcn.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for io.h
-- Looking for io.h - not found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for search.h
-- Looking for search.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for sys/time.h
-- Looking for sys/time.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Performing Test C_HAS_inline
-- Performing Test C_HAS_inline - Success
-- Check size of signed short
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_5c3ac
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_5c3ac
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_5c3ac
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_5c3ac.app/cmTC_5c3ac
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_5c3ac

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:64 (check_type_size)


-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_53722
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_53722
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_53722
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_53722.app/cmTC_53722
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_53722

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:65 (check_type_size)


-- Check size of signed int
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_414e8
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_414e8
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_414e8
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_414e8.app/cmTC_414e8
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_414e8

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:66 (check_type_size)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_ba45d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_ba45d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ba45d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ba45d.app/cmTC_ba45d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_ba45d

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:67 (check_type_size)


-- Check size of signed long
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_5d5b7
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_5d5b7
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_5d5b7
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_5d5b7.app/cmTC_5d5b7
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_5d5b7

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:68 (check_type_size)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_0611f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_0611f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0611f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0611f.app/cmTC_0611f
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_0611f

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:69 (check_type_size)


-- Check size of signed long long
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_05c63
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_05c63
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_05c63
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_05c63.app/cmTC_05c63
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_05c63

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:70 (check_type_size)


-- Check size of unsigned long long
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_7372d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_7372d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_7372d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_7372d.app/cmTC_7372d
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_7372d

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:71 (check_type_size)


-- Check size of unsigned char *
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_CHAR_P.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_f9bd4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_f9bd4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f9bd4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f9bd4.app/cmTC_f9bd4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_f9bd4

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:72 (check_type_size)


-- Check size of size_t
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIZE_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_786e4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_786e4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_786e4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_786e4.app/cmTC_786e4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_786e4

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:76 (check_type_size)


-- Check size of ptrdiff_t
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_PTRDIFF_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_0c8dd
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_0c8dd
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0c8dd
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0c8dd.app/cmTC_0c8dd
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_0c8dd

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:77 (check_type_size)


-- Check size of INT8
-- Check size of INT8 - failed
-- Check size of INT16
-- Check size of INT16 - failed
-- Check size of INT32
-- Check size of INT32 - failed
-- Looking for floor
-- Looking for floor - found
-- Looking for pow
-- Looking for pow - found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for isascii
-- Looking for isascii - found
-- Looking for memset
-- Looking for memset - found
-- Looking for mmap
-- Looking for mmap - found
-- Looking for getopt
-- Looking for getopt - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for setmode
-- Looking for setmode - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for lfind
-- Looking for lfind - found
-- Performing Test HAVE_SNPRINTF
-- Performing Test HAVE_SNPRINTF - Success
-- Check if the system is big endian
-- Searching 16 bit integer
-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_dac39
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_dac39
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_dac39
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_dac39.app/cmTC_dac39
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_dac39

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/TestBigEndian.cmake:32 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_82de9
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_82de9
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_82de9
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_82de9.app/cmTC_82de9
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_82de9

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/TestBigEndian.cmake:37 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_705f3
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_705f3
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_705f3
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_705f3.app/cmTC_705f3
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_705f3

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/TestBigEndian.cmake:44 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


CMake Error at /usr/local/Cellar/cmake/3.10.0/share/cmake/Modules/TestBigEndian.cmake:49 (message):
  no suitable type found
Call Stack (most recent call first):
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Configuring incomplete, errors occurred!
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeOutput.log".
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeError.log".

Cmake 3.9.1

-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- Setting up iPhoneOS toolchain for IOS_ARCH='arm64'
-- iPhoneOS toolchain loaded
-- The CXX compiler identification is AppleClang 10.0.0.10001145
-- The C compiler identification is AppleClang 10.0.0.10001145
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test HAVE_CXX11 (check file: cmake/checks/cxx11.cpp)
-- Performing Test HAVE_CXX11 - Failed
-- Performing Test HAVE_CXX_FSIGNED_CHAR
-- Performing Test HAVE_CXX_FSIGNED_CHAR - Success
-- Performing Test HAVE_C_FSIGNED_CHAR
-- Performing Test HAVE_C_FSIGNED_CHAR - Success
-- Performing Test HAVE_CXX_W
-- Performing Test HAVE_CXX_W - Success
-- Performing Test HAVE_C_W
-- Performing Test HAVE_C_W - Success
-- Performing Test HAVE_CXX_WALL
-- Performing Test HAVE_CXX_WALL - Success
-- Performing Test HAVE_C_WALL
-- Performing Test HAVE_C_WALL - Success
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE
-- Performing Test HAVE_CXX_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_C_WERROR_RETURN_TYPE
-- Performing Test HAVE_C_WERROR_RETURN_TYPE - Success
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WERROR_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WERROR_ADDRESS
-- Performing Test HAVE_CXX_WERROR_ADDRESS - Success
-- Performing Test HAVE_C_WERROR_ADDRESS
-- Performing Test HAVE_C_WERROR_ADDRESS - Success
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_CXX_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT
-- Performing Test HAVE_C_WERROR_SEQUENCE_POINT - Success
-- Performing Test HAVE_CXX_WFORMAT
-- Performing Test HAVE_CXX_WFORMAT - Success
-- Performing Test HAVE_C_WFORMAT
-- Performing Test HAVE_C_WFORMAT - Success
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_CXX_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY
-- Performing Test HAVE_C_WERROR_FORMAT_SECURITY - Success
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS
-- Performing Test HAVE_CXX_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WMISSING_DECLARATIONS
-- Performing Test HAVE_C_WMISSING_DECLARATIONS - Success
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES
-- Performing Test HAVE_CXX_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WMISSING_PROTOTYPES
-- Performing Test HAVE_C_WMISSING_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES
-- Performing Test HAVE_CXX_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES
-- Performing Test HAVE_C_WSTRICT_PROTOTYPES - Success
-- Performing Test HAVE_CXX_WUNDEF
-- Performing Test HAVE_CXX_WUNDEF - Success
-- Performing Test HAVE_C_WUNDEF
-- Performing Test HAVE_C_WUNDEF - Success
-- Performing Test HAVE_CXX_WINIT_SELF
-- Performing Test HAVE_CXX_WINIT_SELF - Success
-- Performing Test HAVE_C_WINIT_SELF
-- Performing Test HAVE_C_WINIT_SELF - Success
-- Performing Test HAVE_CXX_WPOINTER_ARITH
-- Performing Test HAVE_CXX_WPOINTER_ARITH - Success
-- Performing Test HAVE_C_WPOINTER_ARITH
-- Performing Test HAVE_C_WPOINTER_ARITH - Success
-- Performing Test HAVE_CXX_WSHADOW
-- Performing Test HAVE_CXX_WSHADOW - Success
-- Performing Test HAVE_C_WSHADOW
-- Performing Test HAVE_C_WSHADOW - Success
-- Performing Test HAVE_CXX_WSIGN_PROMO
-- Performing Test HAVE_CXX_WSIGN_PROMO - Success
-- Performing Test HAVE_C_WSIGN_PROMO
-- Performing Test HAVE_C_WSIGN_PROMO - Success
-- Performing Test HAVE_CXX_WUNINITIALIZED
-- Performing Test HAVE_CXX_WUNINITIALIZED - Success
-- Performing Test HAVE_C_WUNINITIALIZED
-- Performing Test HAVE_C_WUNINITIALIZED - Success
-- Performing Test HAVE_CXX_WNO_NARROWING
-- Performing Test HAVE_CXX_WNO_NARROWING - Success
-- Performing Test HAVE_C_WNO_NARROWING
-- Performing Test HAVE_C_WNO_NARROWING - Success
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR
-- Performing Test HAVE_C_WNO_DELETE_NON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_CXX_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS
-- Performing Test HAVE_C_WNO_UNNAMED_TYPE_TEMPLATE_ARGS - Success
-- Performing Test HAVE_CXX_WNO_COMMENT
-- Performing Test HAVE_CXX_WNO_COMMENT - Success
-- Performing Test HAVE_C_WNO_COMMENT
-- Performing Test HAVE_C_WNO_COMMENT - Success
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_CXX_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION
-- Performing Test HAVE_C_FDIAGNOSTICS_SHOW_OPTION - Success
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS
-- Performing Test HAVE_CXX_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS
-- Performing Test HAVE_C_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_CXX_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY
-- Performing Test HAVE_C_WNO_SEMICOLON_BEFORE_METHOD_BODY - Success
-- Performing Test HAVE_CXX_FLTO
-- Performing Test HAVE_CXX_FLTO - Success
-- Performing Test HAVE_C_FLTO
-- Performing Test HAVE_C_FLTO - Success
-- Performing Test HAVE_CPU_NEON_SUPPORT (check file: cmake/checks/cpu_neon.cpp)
-- Performing Test HAVE_CPU_NEON_SUPPORT - Success
-- Performing Test HAVE_CPU_FP16_SUPPORT (check file: cmake/checks/cpu_fp16.cpp)
-- Performing Test HAVE_CPU_FP16_SUPPORT - Success
-- Performing Test HAVE_CPU_BASELINE_FLAGS
-- Performing Test HAVE_CPU_BASELINE_FLAGS - Success
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE
-- Performing Test HAVE_CXX_MFP16_FORMAT_IEEE - Failed
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE
-- Performing Test HAVE_C_MFP16_FORMAT_IEEE - Failed
-- Looking for posix_memalign
-- Looking for posix_memalign - found
-- Looking for malloc.h
-- Looking for malloc.h - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of off64_t
-- Check size of off64_t - failed
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32
-- Performing Test HAVE_C_WNO_SHORTEN_64_TO_32 - Success
-- Performing Test HAVE_C_WNO_ATTRIBUTES
-- Performing Test HAVE_C_WNO_ATTRIBUTES - Success
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES
-- Performing Test HAVE_C_WNO_STRICT_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES
-- Performing Test HAVE_C_WNO_MISSING_PROTOTYPES - Success
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS
-- Performing Test HAVE_C_WNO_MISSING_DECLARATIONS - Success
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE
-- Performing Test HAVE_C_WNO_SHIFT_NEGATIVE_VALUE - Success
-- Performing Test HAVE_C_WNO_UNDEF
-- Performing Test HAVE_C_WNO_UNDEF - Success
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH
-- Performing Test HAVE_C_WNO_IMPLICIT_FALLTHROUGH - Success
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for dlfcn.h
-- Looking for dlfcn.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for io.h
-- Looking for io.h - not found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for search.h
-- Looking for search.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for sys/time.h
-- Looking for sys/time.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Performing Test C_HAS_inline
-- Performing Test C_HAS_inline - Success
-- Check size of signed short
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_ccb80
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_ccb80
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ccb80
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ccb80.app/cmTC_ccb80
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_ccb80

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:64 (check_type_size)


-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_f18e1
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_f18e1
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f18e1
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f18e1.app/cmTC_f18e1
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_f18e1

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:65 (check_type_size)


-- Check size of signed int
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_0e115
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_0e115
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0e115
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0e115.app/cmTC_0e115
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_0e115

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:66 (check_type_size)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_3ec16
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_3ec16
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_3ec16
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_3ec16.app/cmTC_3ec16
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_3ec16

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:67 (check_type_size)


-- Check size of signed long
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_43f9e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_43f9e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_43f9e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_43f9e.app/cmTC_43f9e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_43f9e

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:68 (check_type_size)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_cb1f0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_cb1f0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_cb1f0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_cb1f0.app/cmTC_cb1f0
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_cb1f0

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:69 (check_type_size)


-- Check size of signed long long
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_d2d10
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_d2d10
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_d2d10
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_d2d10.app/cmTC_d2d10
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_d2d10

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:70 (check_type_size)


-- Check size of unsigned long long
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_LONG_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_88f5b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_88f5b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_88f5b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_88f5b.app/cmTC_88f5b
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_88f5b

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:71 (check_type_size)


-- Check size of unsigned char *
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_UNSIGNED_CHAR_P.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_b28f2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_b28f2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_b28f2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_b28f2.app/cmTC_b28f2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_b28f2

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:72 (check_type_size)


-- Check size of size_t
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_SIZE_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_42f3e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_42f3e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_42f3e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_42f3e.app/cmTC_42f3e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_42f3e

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:76 (check_type_size)


-- Check size of ptrdiff_t
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/SIZEOF_PTRDIFF_T.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_f275a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_f275a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f275a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_f275a.app/cmTC_f275a
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_f275a

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  3rdparty/libtiff/CMakeLists.txt:77 (check_type_size)


-- Check size of INT8
-- Check size of INT8 - failed
-- Check size of INT16
-- Check size of INT16 - failed
-- Check size of INT32
-- Check size of INT32 - failed
-- Looking for floor
-- Looking for floor - found
-- Looking for pow
-- Looking for pow - found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for isascii
-- Looking for isascii - found
-- Looking for memset
-- Looking for memset - found
-- Looking for mmap
-- Looking for mmap - found
-- Looking for getopt
-- Looking for getopt - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for setmode
-- Looking for setmode - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for lfind
-- Looking for lfind - found
-- Performing Test HAVE_SNPRINTF
-- Performing Test HAVE_SNPRINTF - Success
-- Check if the system is big endian
-- Searching 16 bit integer
-- Check size of unsigned short
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_SHORT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_ccbc4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_ccbc4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ccbc4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_ccbc4.app/cmTC_ccbc4
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_ccbc4

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/TestBigEndian.cmake:32 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned int
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_INT.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_0794e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_0794e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0794e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_0794e.app/cmTC_0794e
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_0794e

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/TestBigEndian.cmake:37 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Check size of unsigned long
CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:114 (try_compile):
  Cannot copy output executable

    ''

  to destination specified by COPY_FILE:

    '/build/build-arm64-iphoneos/CMakeFiles/CheckTypeSize/CMAKE_SIZEOF_UNSIGNED_LONG.bin'

  Unable to find the executable at any of:

    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/cmTC_438e2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Release/cmTC_438e2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_438e2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Debug/cmTC_438e2.app/cmTC_438e2
    /build/build-arm64-iphoneos/CMakeFiles/CMakeTmp/Development/cmTC_438e2

Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/CheckTypeSize.cmake:239 (__check_type_size_impl)
  /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/TestBigEndian.cmake:44 (CHECK_TYPE_SIZE)
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


CMake Error at /usr/local/Cellar/cmake/3.9.1/share/cmake/Modules/TestBigEndian.cmake:49 (message):
  no suitable type found
Call Stack (most recent call first):
  3rdparty/libtiff/CMakeLists.txt:227 (test_big_endian)


-- Configuring incomplete, errors occurred!
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeOutput.log".
See also "/build/build-arm64-iphoneos/CMakeFiles/CMakeError.log".

--- a/platforms/ios/cmake/Toolchains/common-ios-toolchain.cmake
+++ b/platforms/ios/cmake/Toolchains/common-ios-toolchain.cmake
@@ -99,30 +99,27 @@ endif()
 
 if(NOT __IN_TRY_COMPILE)
   set(_xcodebuild_wrapper "${CMAKE_BINARY_DIR}/xcodebuild_wrapper")
-  if(NOT DEFINED CMAKE_MAKE_PROGRAM)  # empty since CMake 3.10
-    find_program(XCODEBUILD_PATH "xcodebuild")
-    if(NOT XCODEBUILD_PATH)
-      message(FATAL_ERROR "Specify CMAKE_MAKE_PROGRAM variable ('xcodebuild' absolute path)")
+  if(NOT EXISTS "${_xcodebuild_wrapper}")
+    set(_xcodebuild_wrapper_tmp "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/xcodebuild_wrapper")
+    if(NOT DEFINED CMAKE_MAKE_PROGRAM)  # empty since CMake 3.10
+      find_program(XCODEBUILD_PATH "xcodebuild")
+      if(NOT XCODEBUILD_PATH)
+        message(FATAL_ERROR "Specify CMAKE_MAKE_PROGRAM variable ('xcodebuild' absolute path)")
+      endif()
+      set(CMAKE_MAKE_PROGRAM "${XCODEBUILD_PATH}")
     endif()
-    set(CMAKE_MAKE_PROGRAM "${XCODEBUILD_PATH}")
-  endif()
-  if(NOT CMAKE_MAKE_PROGRAM STREQUAL _xcodebuild_wrapper)
+    if(CMAKE_MAKE_PROGRAM STREQUAL _xcodebuild_wrapper)
+      message(FATAL_ERROR "Can't prepare xcodebuild_wrapper")
+    endif()
     if(APPLE_FRAMEWORK AND BUILD_SHARED_LIBS)
-      set(_xcodebuild_wrapper_tmp "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/xcodebuild_wrapper")
-      file(WRITE "${_xcodebuild_wrapper_tmp}" "#!/bin/sh
-${CMAKE_MAKE_PROGRAM} IPHONEOS_DEPLOYMENT_TARGET=${IPHONEOS_DEPLOYMENT_TARGET} -sdk ${CMAKE_OSX_SYSROOT} \$*")
-      # Make executable
-      file(COPY "${_xcodebuild_wrapper_tmp}" DESTINATION ${CMAKE_BINARY_DIR} FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
-      set(CMAKE_MAKE_PROGRAM "${_xcodebuild_wrapper}" CACHE INTERNAL "" FORCE)
+      set(XCODEBUILD_EXTRA_ARGS "${XCODEBUILD_EXTRA_ARGS} IPHONEOS_DEPLOYMENT_TARGET=${IPHONEOS_DEPLOYMENT_TARGET} -sdk ${CMAKE_OSX_SYSROOT}")
     else()
-      set(_xcodebuild_wrapper_tmp "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/xcodebuild_wrapper")
-      file(WRITE "${_xcodebuild_wrapper_tmp}" "#!/bin/sh
-${CMAKE_MAKE_PROGRAM} IPHONEOS_DEPLOYMENT_TARGET=${IPHONEOS_DEPLOYMENT_TARGET} ARCHS=${IOS_ARCH} -sdk ${CMAKE_OSX_SYSROOT} \$*")
-      # Make executable
-      file(COPY "${_xcodebuild_wrapper_tmp}" DESTINATION ${CMAKE_BINARY_DIR} FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
-      set(CMAKE_MAKE_PROGRAM "${_xcodebuild_wrapper}" CACHE INTERNAL "" FORCE)
+      set(XCODEBUILD_EXTRA_ARGS "${XCODEBUILD_EXTRA_ARGS} IPHONEOS_DEPLOYMENT_TARGET=${IPHONEOS_DEPLOYMENT_TARGET} ARCHS=${IOS_ARCH} -sdk ${CMAKE_OSX_SYSROOT}")
     endif()
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/xcodebuild_wrapper.in" "${_xcodebuild_wrapper_tmp}" @ONLY)
+    file(COPY "${_xcodebuild_wrapper_tmp}" DESTINATION ${CMAKE_BINARY_DIR} FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
   endif()
+  set(CMAKE_MAKE_PROGRAM "${_xcodebuild_wrapper}" CACHE INTERNAL "" FORCE)
 endif()
 
 # Standard settings
@@ -130,11 +127,11 @@ set(CMAKE_SYSTEM_NAME iOS)
 
 # Apple Framework settings
 if(APPLE_FRAMEWORK AND BUILD_SHARED_LIBS)
-  set(CMAKE_SYSTEM_VERSION 8.0)
+  set(CMAKE_SYSTEM_VERSION "${IPHONEOS_DEPLOYMENT_TARGET}")
   set(CMAKE_C_SIZEOF_DATA_PTR 4)
   set(CMAKE_CXX_SIZEOF_DATA_PTR 4)
 else()
-  set(CMAKE_SYSTEM_VERSION 6.0)
+  set(CMAKE_SYSTEM_VERSION "${IPHONEOS_DEPLOYMENT_TARGET}")
   set(CMAKE_SYSTEM_PROCESSOR "${IOS_ARCH}")
 
   if(AARCH64 OR X86_64)

--- a/platforms/ios/cmake/Toolchains/xcodebuild_wrapper.in
+++ b/platforms/ios/cmake/Toolchains/xcodebuild_wrapper.in
@@ -1,0 +1,19 @@
+#!/bin/sh
+
+# Force 'Debug' configuration
+# Details: https://github.com/opencv/opencv/issues/13856
+if [[ "$@" =~ "-project CMAKE_TRY_COMPILE.xcodeproj" && -z "${OPENCV_SKIP_XCODEBUILD_FORCE_TRYCOMPILE_DEBUG}" ]]; then
+  ARGS=()
+  for ((i=1; i<=$#; i++))
+  do
+    arg=${!i}
+    ARGS+=("$arg")
+    if [[ "$arg" == "-configuration" ]]; then
+      ARGS+=("Debug")
+      i=$(($i+1))
+    fi
+  done
+  set -- "${ARGS[@]}"
+fi
+
+@CMAKE_MAKE_PROGRAM@ @XCODEBUILD_EXTRA_ARGS@ $*

